app/code/Magento/Catalog/etc/di.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0"?>
 <!--
 /**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
+ * Copyright 2013 Adobe
+ * All Rights Reserved.
```

Comment:
why this change? copyrighted code should not be merged in this repo

Reply:
@fballiano, see the thread in https://github.com/magento/magento2/pull/39099#pullrequestreview-2399484952

Reply:
No idea 🤷, you can take it up with @sidolov, he's on [Magento's Engineering Slack workspace](https://developer.adobe.com/open/magento/slack) if you want to discuss it in more detail.